python3/emailmessage/email_sample_script.py

```diff
@@ -540,7 +540,7 @@ def addBasicInfoToIncident(self):
         No return value.
         """
         newReporterInfo = emailmessage.sender.address
-        if emailmessage.sender.name is not None:
+        if hasattr(emailmessage.sender, "name"):
```

Comment:
I think you still want to check that name is not none.
```suggestion
        if hasattr(emailmessage.sender, "name") and emailmessage.sender.name is not None:
```